### DISCUSSION TITLE
Feature Idea - Integrate with Paperless-ngx

### DISCUSSION THREAD

**Discussion by philosowaffle** [hay-kot/homebox#58] [Ideas]
Homebox has a little bit of feature overlap with [paperless-ngx](https://github.com/paperless-ngx/paperless-ngx) specifically when it comes to tracking documents such as receipts, warranties, and even manuals.

Instead of having to fully re-invent the wheel, I wonder if Homebox could get a more feature rich document experience faster by interfacing with the Paperless [api](https://paperless-ngx.readthedocs.io/en/latest/api.html)?

**Comment by kbekus** [comment]
This would be cool.  I hate to be that guy, but even integrating with Nextcloud as well - using Nextcloud as the repository for files, pictures and other attachments while using Homebox for the rich metadata. This would be a great Nextcloud plugin.

**Comment by katosdev** [comment]
Love this idea, and I can very much see a use case. I’d be willing to help crowdfund this also.
